serverless.yml

```diff
@@ -23,7 +23,7 @@ functions:
     handler: src/functions/updateUploadStatus/framework/handler.handler
     events:
       - http:
-          path: test-result/{app-ref}/upload
+          path: test-result/{appRef}/upload
```

Comment:
Are you sure that's the correct param name? and not app-ref?

Reply:
current terraform code: 
```terraform
# test-results/{app-ref}/upload PUT #####################################################################
resource "aws_api_gateway_resource" "private_test_results_appref" {
  rest_api_id = "${aws_api_gateway_rest_api.api_private.id}"
  parent_id   = "${aws_api_gateway_resource.private_test_results.id}"
  path_part   = "{app-ref}"
}
````

Reply:
Yeah didn't realise there was extra infra work required if I wanted to change the param name. The current path param won't allow the lambda to be run locally. Will hold off for now until a decision is made.